client/src/components/ProfileEdit/index.js

```diff
@@ -260,29 +226,47 @@ const ProfileEdit = function () {
         WHERE YOU LIVE
       </InputText>
 
-      {/*GENDER*/}
-      <Grid container item className={classes.input}>
-        <Grid item xs={4} md={3}>
-          <Label id="gender">GENDER</Label>
-        </Grid>
-        <Grid item xs={8} md={5}>
-          <TextField
-            id="gender"
-            name="gender"
-            value={values.gender}
-            onChange={handleInputChange}
-            select
-            className={classes.select}
-            variant="outlined"
-          >
-            {genders.map((value) => (
-              <MenuItem value={value} key={value}>
-                {value}
-              </MenuItem>
-            ))}
-          </TextField>
+      {/*HOURLY RATE*/}
+      {profile && profile.isSitter && (
```

Comment:
This and l252 can probably be wrapped together in case anything chances with the variables